src/Microsoft.Health.Fhir.Core/Features/KnownQueryParameterNames.cs

```diff
@@ -69,5 +69,9 @@ public static class KnownQueryParameterNames
         public const string AnonymizationConfigurationFileEtag = "_anonymizationConfigEtag";
 
         public const string OutputFormat = "_outputFormat";
+
+        public const string TypeFilter = "_typeFilter";
+
+        public const string IncludeAssociatedData = "includeAssociatedData";
```

Comment:
> public const string IncludeAssociatedData = "includeAssociatedData"; [](start = 7, length = 69)

I can't find any usage of this.

Reply:
Sorry, left over from testing a different feature. Will remove.